node/test/PackageSourceTests.ts

```diff
@@ -17,6 +17,40 @@ const testSchemaB = "org.opent2t.test.schemas.b";
 const testTranslatorOne = "org.opent2t.test.translators.one";
 const testTranslatorTwo = "org.opent2t.test.translators.two";
 
+const testOnboardingPackageOne = "opent2t-onboarding-org-opent2t-test-onboarding-one";
+const testOnboardingOnboardingOne = "org.opent2t.test.onboarding.one";
+const testOnboardingSchemaOne = "org.opent2t.test.onboarding.one";
+
+
+test("Get onboarding One info", async t => {
+    let packageInfo: PackageInfo | null = await testPackageSource.getPackageInfoAsync(testOnboardingPackageOne);
```

Comment:
Because module resolution in Node.js is normally done in terms of a package name + a module name (relative path to the module within the package). The PackageSource base class declares methods that provide module resolution in those terms. LocalPackageSource extends the PackageSource base, but it has to work with directories that unfortunately aren't properly structured as packages.

---
In reply to: [75347621](https://github.com/openT2T/opent2t/pull/22#discussion_r75347621) [](ancestors = 75347621)